Jenkinsfile

```diff
@@ -100,6 +100,13 @@ podTemplate(cloud: 'openshift', label: 'coreos-assembler', yaml: pod, defaultCon
                 utils.shwrap("""
                 coreos-assembler buildprep s3://${s3_builddir}
                 """)
+            } else if (!prod && utils.path_exists("/srv/devel/${devel_prefix}/build")) {
+                // XXX: should integrate this into buildprep
```

Comment:
i.e. buildprep could accept a file:/// URI?

Reply:
Right, exactly. Would be trivial, just didn't want to distract away from getting pipelines in.